cli/cmd/get.go

```diff
@@ -498,6 +502,23 @@ func getAPI(env cliconfig.Environment, apiName string) (string, error) {
 	return realtimeAPITable(apiRes, env)
 }
 
+func apiHistoryTable(pastDeploys []schema.PastDeploy) string {
+	t := table.Table{
+		Headers: []table.Header{
+			{Title: "api id"},
+			{Title: "last deployed"},
```

Comment:
note to reviewer: I don't love this header, but nothing jumped out to me that was obviously better, let me know what you think